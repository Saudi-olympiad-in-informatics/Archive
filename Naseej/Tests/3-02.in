42397 736
35644 24583 889607879
20159 7187 87423143
38826 13466 501273878
13137 36659 714538436
12091 16026 496415816
12656 10708 765795996
32691 28884 858605972
35050 27437 916687575
24396 28360 306948719
16017 27228 707137873
24800 23069 379225356
28005 35163 317671204
5945 15891 243630792
10702 1937 595950332
6773 40973 815966319
29916 11629 229726636
23932 19762 819039659
34562 7093 617531418
7319 721 847204153
26596 5416 198054152
21827 13370 713949695
27448 1095 688842088
8223 7886 578243958
6172 18713 898612555
15315 39524 905864833
14102 23952 476166000
11425 8785 108395918
5208 24082 982842346
3489 20861 88341840
19012 19220 477907623
19177 34092 183397206
32951 2657 590407771
6886 24894 715789753
19536 14521 277154986
20182 2273 258106050
5418 42041 987818949
23581 16306 102874620
9014 18848 846504752
35459 32292 178253190
32471 17775 893241359
4980 34739 937672704
31690 10210 136027977
25652 7221 636419692
18001 6286 748915941
26837 19553 662560732
37548 13414 607067406
20688 24219 535578776
21941 9902 7016473
11479 12207 671057632
9114 31874 445801483
29983 23813 515192728
9638 22360 966962060
16251 13059 138245146
4435 15988 101483079
26095 34182 272412237
38205 33830 251090392
37869 37825 557934720
25782 21135 126020977
41734 22858 315315527
24942 22061 931367412
1498 18034 616503690
1689 38142 167554183
13024 35746 7995375
14484 12491 541630372
34654 2230 24182427
1653 37778 507313341
4961 7180 221028655
25866 33008 695629941
34417 29479 878759451
13213 7142 157056662
28570 4606 619583924
17939 40941 401925505
27769 20441 289631133
17959 16142 233111778
14324 595 958319388
7517 14154 821239403
37976 89 436519173
38570 13637 841469267
9476 37404 742740278
35661 36039 471107734
893 33853 869540252
33073 34220 23589430
7086 36093 65952014
40057 20018 200199095
5765 34675 260290370
30812 18567 2502239
4373 3300 844960167
36564 22451 407041416
20648 32155 739935289
39616 1206 628114614
15 12206 918356246
13535 40227 599994072
13558 21460 302234959
662 31699 192112165
30615 31836 392267519
2522 13497 470447048
25057 21100 715898346
10187 27686 283093331
9965 11292 291838114
23957 132 786118124
18348 40231 105767264
39800 2296 481711262
38275 9331 104083953
22472 7619 165623987
42136 35864 495822512
23694 8626 371470541
30372 33480 570531896
18538 28018 147062357
11320 9797 245572960
12139 5562 524943497
39857 30670 507795922
6175 33419 902854066
8286 27356 479098924
34575 989 600447529
8341 12065 684440965
3978 483 691785913
17571 35408 471155957
32098 30218 506994162
35494 20456 991657257
32734 29894 719278372
20675 35672 345401617
34938 8778 855957641
32440 28286 137459025
14210 24474 536911527
4353 33725 741275719
10142 40307 468343052
15086 36926 78180178
27873 21096 876021181
42043 1358 853783122
19001 13526 366731268
3227 3439 258948461
32356 17272 345636793
3808 5695 811873186
5017 16315 826559275
34648 3383 105272955
1386 26816 245633494
1758 31809 175668903
502 21199 337858866
41019 7514 631401493
40149 1371 972881373
4884 22477 903278043
33683 5550 63207617
5577 29194 319484900
11426 6410 435409579
28658 3420 698106912
7834 26976 426448057
3135 30074 970546169
11049 38492 146238049
4985 11755 955867792
26065 24683 690063275
13673 23655 487189903
18406 19477 620934789
6380 32571 36203733
15390 27457 718711809
3242 10257 34858007
35247 4448 278111249
5420 18282 823806722
36095 29359 652005452
30932 1846 234717024
13868 36214 138743978
20820 11683 297737160
33160 2491 540323088
31062 10965 47660053
24160 30051 472576999
23743 9325 74009620
34268 25590 241216457
16194 32865 298390322
15344 3036 952100455
40961 30526 563242236
6728 38349 208918821
25802 34397 505962506
3344 14449 98870539
12563 5366 25666462
40946 20475 751188358
13440 12544 489742911
28265 8917 845372448
8033 11973 331464825
35315 37811 274130102
12526 18545 572721792
7280 1032 294064595
22182 18160 412078164
30273 30821 552462259
11400 17908 918660281
6798 36207 203364225
16034 18592 968550639
11622 38344 300014958
28566 22461 479550488
19107 4962 300271046
6433 27049 496459091
23099 22285 251550203
16517 15752 853577159
11288 20350 943038428
9128 13210 210589008
31504 17785 119023010
7208 14456 778087989
42213 30788 820695120
9446 1171 122714856
14426 22216 423518968
9734 7973 980272577
19194 30739 807439779
4789 12448 265137687
1632 34570 709890731
23237 920 197782235
41383 20380 831586959
7542 34428 73079650
10827 25220 835397292
29847 24043 198485656
34141 19587 593008961
39251 31056 562672997
11673 16022 106961932
24738 30146 942115898
23391 41410 23337143
20187 18129 399381798
8504 11842 366597793
23392 21822 423844859
35290 13076 371261885
35360 36177 46570254
25321 12258 622460094
9235 28318 93779823
41996 39284 299455607
35549 26069 616603981
36476 15858 772798495
40192 10413 22503436
9966 22096 492763223
13432 12681 40708863
20822 41550 202755983
7373 16408 913934420
9381 12281 65054407
9713 6235 714443917
8426 14530 679022405
41291 25080 398780902
15392 8585 792550061
20626 7774 436667630
22802 40392 129439415
15837 6968 473518755
7307 6371 727696180
35824 21206 646880608
33777 6089 132885923
894 40572 823372097
20998 38010 370260108
8961 35729 317460226
40930 10085 480891046
7739 12389 775538997
10575 34388 649725052
11500 42164 447322938
29171 5516 630522621
31104 34680 80418552
22493 34770 475978442
469 5753 378735482
29004 16405 186267712
31628 4268 623986548
31431 9500 720135862
1692 13684 521218574
18488 28256 571798817
2453 26408 469571383
23117 6969 850337398
14498 9772 133188932
27048 2156 565495646
25249 2596 524822318
40635 29753 193701723
14815 42386 333011826
42215 28901 315478287
38863 39997 366850816
1467 26223 799889953
40108 40270 337034818
13495 32556 95943175
36918 18942 150942323
8327 26504 206715052
36751 30368 671365635
14556 18053 53969421
11247 11691 301490974
5289 2966 933448291
21379 13056 894774036
13726 39905 349052784
32878 21356 660993216
34600 25601 885768268
13355 16456 881083092
34958 32136 289924312
17132 37939 49414856
21864 11773 773103431
23158 20340 635101787
39548 15189 411478301
7742 2589 796675647
10749 5697 254456719
23181 1128 79156363
27716 4929 639616742
38292 32061 580326096
9393 28958 902092875
22833 28739 631826459
4206 33437 513916045
22055 19204 211173049
13790 41859 757806486
20442 21109 652463978
22105 25955 355512094
14226 2441 763295458
34821 3590 751770557
19841 35434 190222719
15666 12861 635792381
17195 15242 225300970
29611 12063 709219730
19283 15078 114792876
18955 36212 532894193
10218 6073 575658630
39297 40659 111934107
41928 34712 421959098
30116 40048 839672417
8976 16485 856248812
36770 16956 750844053
32585 31403 558530603
30624 5632 344836790
1088 33627 664649473
17586 19869 37550016
7367 16252 160381159
12184 36861 53445926
37748 28671 246841193
26102 35103 451105869
5649 9489 271352341
6640 10798 911032824
14666 32797 867350240
9862 8687 746139550
23365 4811 452683795
36145 5237 205602411
22282 19014 272700910
13615 7874 722690390
12171 10234 296054297
24006 12398 449402523
41258 16990 243861109
41634 10482 952212457
5436 11779 951551527
37068 37758 516124605
28325 13622 833760615
39986 37524 31687433
39365 35473 339337124
39108 26523 209527637
24541 5345 121355237
26136 20969 222651596
17322 28501 810657637
20396 40376 448987237
27898 4167 955181548
33816 21581 981014263
32520 40087 953930975
22440 40120 476608121
37816 18019 252768385
37253 17935 750977228
30656 12455 623317497
36631 29346 701257969
7962 37462 383249275
25195 37485 7500255
19962 7536 503749777
18636 3110 25858786
39964 737 5579067
16563 21286 951589312
25119 32587 484412315
24128 20106 411097839
25061 13197 148066618
29935 25655 546845413
4063 37032 375160013
17796 7296 936720540
578 32727 924301053
2908 22677 746259736
3510 27028 371192708
41586 3854 570210269
20105 27148 853413512
40566 41799 563943016
14898 796 441703222
37577 9322 805164131
23842 9675 263264897
26430 2243 324823045
3120 7990 239943762
28270 27695 540781598
42045 16092 9629892
11843 34587 129955114
1436 26948 314903748
21538 2736 700182984
13862 11126 544417130
37272 11103 332101254
23128 29086 729027020
14604 37727 517187293
16933 7938 530906545
25056 36912 116671448
2591 12513 620176726
290 9643 306433137
39350 11193 816561706
33057 8128 351419871
465 12978 434695193
608 41954 200547682
34467 29215 963809512
21736 35457 407700246
11495 14220 336872359
40284 26625 815024827
17010 30826 768215903
29569 27826 586652368
25561 20994 44897331
1519 30545 675284947
32043 38568 476575851
36660 37738 317072339
18291 18841 106185066
2497 33614 926103078
32546 8544 634490217
10079 25578 771737228
3133 24421 631099962
38996 10144 281445673
11992 20383 563951546
4483 19898 258023573
13269 550 818954813
14502 35619 689982183
6071 35552 255632719
4019 35532 673589600
10173 2041 410864199
41224 40681 477828732
38495 34362 945215979
11624 2792 632902459
9049 41557 253007375
36232 32293 106033046
35433 25273 745913889
29477 32786 92727994
2271 632 767173193
18244 20981 695687949
17135 14717 827837174
12640 10408 699586002
12665 29017 584118808
13635 12631 503957897
41745 28228 500203225
34760 16314 247834423
11207 17577 743757364
23415 24191 650242655
34993 7443 109428034
30485 22162 20675130
37533 19102 948260522
22561 37470 398505200
22758 8668 252848446
32360 18340 168104152
10460 25714 804965209
26291 23723 804176617
15777 23563 89382266
5532 8649 445408658
40563 37185 7020968
14844 41969 626981089
15363 24283 502200438
9346 41835 910367627
34926 1146 578026659
21057 11384 279250532
38346 1433 57068832
21860 15036 470561554
31264 30379 220639916
35147 31471 899650898
4201 23260 72136596
16919 11059 775229925
38219 33824 725928433
5205 33416 648635047
7374 32411 729555356
36620 16806 964138460
8952 3012 830981934
1062 40135 89721571
28429 24811 716335166
19079 41919 177575777
13339 16825 414439838
36943 20740 918586938
26547 31970 838507839
21124 20521 899300752
2222 24115 182838726
19709 10990 760226985
28836 4837 846907482
17406 9793 688464726
27653 27717 915160926
9432 21505 828198021
35374 18297 570370892
3687 19917 821346068
29089 5471 516454805
36728 37880 362674663
12506 3335 552526131
38745 25297 650600449
35287 18430 823954422
31407 12307 428987727
37798 19131 550522640
42093 1840 883324334
10875 7513 35866367
32091 16230 501595062
34740 39416 872689771
25384 19881 613416229
2060 17344 770596137
30202 976 367854620
29123 22481 266687385
35386 38135 554282592
2742 25803 168455630
5719 6660 410331303
15518 22600 898161301
32435 13081 870076350
22591 41865 244871274
17183 9948 512258853
18832 9092 32206175
9544 20731 140330233
18570 16740 502433890
22403 19818 86206596
2849 5840 518686448
12873 23536 855049830
14661 38480 875235724
1359 21075 354632368
31814 37212 339777989
2456 4286 441831893
29765 15934 881985792
5728 3594 191758671
20833 29114 716121367
20133 26222 60037215
5467 29098 192079975
36165 9563 630513265
34639 40141 630269333
3731 34198 982572154
20926 35016 576276647
18182 1351 854194106
25136 16430 279913073
31008 1001 1883528
16737 22714 58070543
32224 18040 805603015
19210 32892 55059123
22584 26196 917391171
19824 2876 871732748
34832 26824 19028382
7482 38954 487300253
28460 31066 421724170
25857 31878 659449615
42044 1873 198909585
35238 20578 777820130
28046 6317 480772076
22245 20183 806490108
17689 4656 586673737
40644 32333 501270048
2364 20656 567045432
21475 404 227743044
6927 13108 23659170
5845 23014 233747623
36547 29997 92322603
25198 38208 812150364
20016 17489 77581606
11977 2860 602079930
38489 21887 539986132
20057 36995 283536107
32846 2259 20084517
22421 22954 895134752
1018 6668 689193232
7860 1941 677655528
17215 13483 445483245
25450 35094 188425651
20347 35570 732927572
14303 36126 402366008
32165 14615 90035815
1004 12277 831314131
3060 6920 842716122
22133 5565 855421389
26615 4586 57295750
2993 5351 648431572
9753 644 380542156
6972 32226 625746450
32572 19174 458760853
34018 40038 512117402
24847 30290 536772948
27800 13109 409728442
25409 11265 613111162
21392 33535 772450563
6800 17343 47580275
35441 35900 275666471
15440 31442 266698563
204 7177 311184166
7842 16696 999779782
38295 28098 605949882
5065 36950 760979234
9307 1473 242884626
22557 19660 516878714
15565 26052 611817388
2997 6290 953096463
3297 1868 263479997
16894 226 589851724
34298 1537 251958383
16253 40527 657279193
12928 18037 59754656
24687 11153 739965436
19939 19496 765994906
41245 23187 352961089
23295 11302 576284418
26903 22629 688130466
10016 25407 903042660
11550 16455 502231242
39576 22882 660452863
32974 34523 940502871
22508 36073 433470337
25676 9784 125519157
27514 14342 141586546
37699 20465 860239079
13080 34690 461062574
23078 11775 640105513
27954 28778 444000210
14799 32765 612134867
16688 17337 912827425
23850 21552 290871955
15460 10743 422304236
36590 21671 384696514
391 33880 400075256
20635 38432 547187287
39591 26362 873248142
11064 19504 120206559
7166 4873 356456558
29682 31598 342120816
17192 30616 310819221
30716 21846 865006252
1341 37327 420823328
32071 8506 901943638
41559 24925 247598063
7554 25442 531101331
2995 36287 345743334
26183 30098 149094587
5495 39422 909345340
39199 961 847356878
31642 7488 838599586
30285 3250 97489905
37255 14071 96995116
27640 40672 898278205
25516 41940 7967804
2838 23380 320367200
8569 12199 468496966
28333 35696 154784607
35797 14112 974242811
30579 21790 725274809
11765 41405 203173383
19146 41387 862988376
15948 5152 39059565
22100 29726 757569527
41 37252 260423699
9595 14675 220850087
38881 28600 321792880
17573 6472 348256544
12597 19765 45524456
4374 41250 142090723
20439 25377 709800466
12516 41358 795603803
13640 24125 720313501
42007 33992 436305532
27135 17488 121310891
34271 2582 842375278
14050 26886 441797819
19532 33834 26006657
10322 29880 867755393
6448 15729 511756452
6034 22240 12783580
28759 33189 706883944
13809 31495 734569392
34715 40085 523470038
42163 32631 751485465
27701 37331 602227348
15352 9650 445505588
22304 30710 167957357
3595 20977 750880939
11470 34830 944324514
12112 26764 190198291
25867 12463 159055265
20669 815 494906910
42265 2406 374245804
35402 34274 764301153
8866 20579 71287433
38855 40041 561780355
26422 41515 964841865
25602 18633 563124147
3760 38798 599848219
18509 16558 869647980
41119 27000 964093491
23470 5096 908306488
17750 28995 55023277
16574 42340 987231768
35006 30987 494312117
6847 13392 299836426
28236 3006 743576061
27634 42289 599654817
13932 18762 496115244
25502 25227 524371599
27832 316 298139623
34553 21130 294023988
3102 14920 274682622
18131 6952 694838318
15241 41288 146129245
32971 25009 11325278
29355 22943 240778779
32755 13944 169867720
31258 15402 638817805
7617 33224 315404385
15229 34749 947813181
41503 23942 256676483
31108 17995 600816968
4571 11120 816485217
17906 32442 978645343
18424 515 728884325
30032 5186 411300118
15702 18156 205247244
6572 33103 197467548
37188 10434 723299702
26159 24199 558521850
30105 11151 808753539
33153 41749 225711146
6533 39373 697577768
34436 40932 530247843
35942 19958 684767838
20039 5001 458637886
6477 38520 958126240
17574 7535 42587702
10942 40184 240629494
24695 12709 190014978
2891 19796 464419981
33987 9020 212510780
40929 20061 763512387
8582 24260 970146697
31780 35036 701179705
17549 12071 402015476
17982 4739 880979375
23400 11200 410630734
22958 9357 419671875
15912 36290 669459789
37437 2145 731433301
1652 9376 895469243
17821 11815 999115127
1874 8204 152166969
8527 27444 159294226
18069 20113 558936165
16340 34109 287044051
21260 31117 358440545
15191 4171 720204461
2437 16878 67950855
26322 30490 850101681
3574 9067 657704221
24213 25757 449566503
33198 2279 539751871
24608 34382 93091937
26088 4907 360049451
23966 9994 202021290
7338 32525 643600948
27081 8146 675835368
2165 28663 640676925
37163 34517 263650486
14314 40429 800883573
