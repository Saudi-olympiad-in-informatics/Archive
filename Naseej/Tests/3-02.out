47267074246282